### ISSUE TITLE
Unified inbox not working - version 5.206

### ISSUE THREAD

**Issue by p166821** [k9mail/k-9#2509]
Please search to check for an existing issue (including closed issues, for which the fix may not have yet been released) before opening a new issue: https://github.com/k9mail/k-9/issues?q=is%3Aissue

### Expected behavior
Tell us what should happen

I had 5 emails in K9 and it should have aggregated everything in unified inbox

### Actual behavior
Tell us what happens instead

Many latest emails are omitted in unified inbox, but some of the outdated ones are still on top of unified inbox

### Steps to reproduce
1.check messages universally
2.compare latest msgs from individual inbox to Global inboxes.
3.

### Environment
K-9 Mail version:5.206

Android version: 7.0 (no recent updates between k9 update

Account type (IMAP, POP3, WebDAV/Exchange):
All imap

**Comment by p166821** [comment]
Thank you for your help!

**Comment by pshem** [comment]
Do they show up in All messages? And do they show in Unified Inbox at all, for example on the bottom?
a) would mean you've sent them,
b) would mean timestamp error/you only recently pulled them from a server, but they're actually older
Or they could be treated as spam by k-9, which should omit them from both individual and unified Inbox, but that doesn't match your description.

**Comment by p166821** [comment]
a)      Thanks for the suggestion – all messages shows the same incomplete sequence of emails. Checked the Unified Inbox and I didn’t find anything at the bottom – chronologically ordered to have the latest msgs first. Some omitted messages are from a new thread. Not every message I replied are showing in unified inbox either…



b)      Yes it’s pretty weird that it’s OK in individual email folder (I don’t push/pull any folders other than Inbox folders), and it’s like “selectively showing” in the unified inbox. I tried to spot a trend but it has both read/unread messages from each of my inbox.

c)       I agree that spam marker would’ve let K9 omit it in both individual and unified inboxes, but it isn’t happening.

d)      I’m trying to enable debugging/logging but it seems to be the first time this issue came up. I thought about rolling back to an older version, but I worried if the settings file may not work properly..
